javaCore/src/main/java/com/hamdan/forzenbook/java/core/GlobalConstants.java

```diff
@@ -0,0 +1,16 @@
+package com.hamdan.forzenbook.java.core;
+
+
+public final class GlobalConstants {
+    public static int EMAIL_LENGTH_LIMIT = 64;
+    public static String LOGIN_BASE_URL = "https://forzen.dev/api/";
+
+    private static GlobalConstants instance = null;
+
+    public static GlobalConstants getInstance() {
```

Comment:
This isn't doing anything. You don't have any instance variables in this class. You can just access it all statically and remove the singleton instance.